### DISCUSSION TITLE
Synch user accounts across grid

### DISCUSSION THREAD

**Discussion by PatRussell87** [Security-Onion-Solutions/securityonion#7864] [Unsupported Versions] [answered]
Do user accounts synch across all nodes in the grid? I have a distributed install. One node that was replaced doesn't seem to have the same user accounts as the rest of the nodes. This is a sensor node. I've been able to manually add a user and change the password but expected it to be automatic.

**Comment by dougburks** [comment, marked as answer]
Assuming you are referring to user accounts at the operating system level then, no, those are not replicated across the grid.

**Comment by PatRussell87** [comment]
That's exactly what I was asking about. Thanks for clearing it up for me!
